src/APIBlueprintImporter.js

```diff
@@ -1,8 +1,19 @@
 import {Fury} from 'fury';
-import {detect} from 'fury-adapter-apib-parser';
-import {parseSync} from 'drafter.js';
+import APIBlueprintParser from 'fury-adapter-apib-parser';
 import APIElementImporter from './APIElementImporter.js'
 
+function promisify(func, thisArg) {
```

Comment:
Promisify is a tempoary solution until async/await is added to Fury, I think this will happen in the next release. Internal refactoring is already done for Fury.

I am writing promisify myself because this is a Paw extension which uses JavaScriptCore. We are not running on Node in the extension and thus node utils module which contains promisify isn't available.